content/24.vaccines-app.md

```diff
@@ -1,11 +1,9 @@
-## Appendix: Vaccine Development Strategies for SARS-CoV-2
-<!-- It would be ideal to loosely organize these sections in the order they appear in the main text, or even split the appendix into a traditional and novel vaccines appendix. -->
+## Appendix
```

Comment:
We should keep some form of title for the appendix for the Manubot HTML table of contents